src/osd/ECBackend.cc

```diff
@@ -275,19 +275,20 @@ struct OnRecoveryReadComplete :
 
 struct RecoveryMessages {
   map<hobject_t,
-      ECBackend::read_request_t> reads;
+      ECBackend::read_request_t> recovery_reads;
   map<hobject_t, set<int>> want_to_read;
-  void read(
+  void recovery_read(
```

Comment:
i'd suggest s/grep/[lsp](https://langserver.org/)/, if you are not using it yet. personally, i've been using both clangd and ccls along with my editor's integrations. they work great under most circumstances.